Appraisals

```diff
@@ -29,36 +11,36 @@ appraise "rails52" do
 end
 
 appraise "rails60" do
-  gem "rails", ">= 6.0.0", "< 6.1"
+  gem "rails", "~> 6.0.6"
   gem "mysql2", ">= 0.4.4"
   gem "pg", ">= 0.18", "< 2.0"
   gem "sqlite3", "~> 1.4"
 end
 
 appraise "rails61" do
-  gem "rails", ">= 6.1.0", "< 6.2"
+  gem "rails", "~> 6.1.7"
   gem "mysql2", ">= 0.4.4"
   gem "pg", ">= 1.1", "< 2.0"
   gem "sqlite3", "~> 1.4"
 end
 
 appraise "rails70" do
-  gem "rails", ">= 7.0.0", "< 7.1"
+  gem "rails", "~> 7.0.8"
   gem "mysql2", ">= 0.4.4"
   gem "pg", ">= 1.1"
-  gem "sqlite3", ">= 1.4"
+  gem "sqlite3", "~> 1.4"
 end
 
 appraise "rails71" do
-  gem "rails", ">= 7.1.0.beta1", "< 7.2"
+  gem "rails", "~> 7.1.3"
   gem "mysql2", ">= 0.4.4"
   gem "pg", ">= 1.1"
-  gem "sqlite3", ">= 1.4"
+  gem "sqlite3", "~> 1.4"
 end
 
-appraise "rails80" do
```

Comment:
so in the changelog it says "Add rails 8 support" and now we remove the appraise `rails80` ?